services/data-info/src/data_info/routes/stats.clj

```diff
@@ -9,6 +9,8 @@
 
 (defroutes* stat-gatherer
 
+            ; FIXME Update metadactyl exception handling when data-info excptn hndlg updated
+            ; metadactyl catches exceptions thrown from this EP.
```

Comment:
The comment is mostly to bring attention to myself that some of the code I've left unchanged will have to be updated. I'd like to think that I'd be diligent, but updating the comment is a good idea.